packages/web/src/scss/components/Tooltip/README.md

```diff
@@ -3,32 +3,22 @@
 Bare Tooltip HTML:
 
 ```html
-<div class="Tooltip" data-spirit-placement="top">
-  Hello there!
-  <span class="Tooltip__arrow"></span>
+<div class="Tooltip" data-spirit-element="tooltip-wrapper">
```

Comment:
Shouldn't we also rename the `tooltip-wrapper`

Reply:
renamed to 'tooltip'